R/poisdm.R

```diff
@@ -0,0 +1,103 @@
+#' Poisson DataMaker
+#'
```

Comment:
needs a brief description of what the function accomplishes, eg "Simulates groups of Poisson-distributed count data."